compiler/x/codegen/OMRTreeEvaluator.cpp

```diff
@@ -6117,13 +6117,79 @@ OMR::X86::TreeEvaluator::iexpandbitsEvaluator(TR::Node *node, TR::CodeGenerator
 TR::Register*
 OMR::X86::TreeEvaluator::mAnyTrueEvaluator(TR::Node *node, TR::CodeGenerator *cg)
    {
-   return TR::TreeEvaluator::unImpOpEvaluator(node, cg);
+   TR::Node *maskNode = node->getFirstChild();
+   TR::Register *maskReg = cg->evaluate(maskNode);
+   TR::Register *resultReg = cg->allocateRegister();
```

Comment:
Other evaluators in this PR have an assert for 32-bit.  Should this one have one too for consistency?